### PR TITLE
styles

### PR DESCRIPTION
**Pull request by kusholhuq**
style

### CODE REVIEW

File: server/public/styles.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.push-right{
+position: relative;
+left: 1.2rem;
+}
```
Comment (thebearingedge) [inline server/public/styles.css]: You probably meant to add left margin here. Bootstrap has spacing helpers.

https://getbootstrap.com/docs/4.4/utilities/spacing/#notation

File: client/components/gradeForm.jsx [hunk 2]
```diff
@@ -48,22 +48,34 @@ class GradeForm extends React.Component {
...
     return (
       <form onSubmit={this.handleSubmit}>
+        <div className='row'>
+          <div className='m-1'>
+            <div className='mb-3'><i className='fas fa-user'></i></div>
+            <div className='mb-3'><i className='fas fa-list'></i></div>
+            <div className='mb-2'><i className='fas fa-graduation-cap'></i></div>
+          </div>
+          <div>
+            <div className='mb-2'><input type='name' placeholder='Name' onChange={this.handleName} value={this.state.name}></input></div>
```
Comment (thebearingedge) [inline client/components/gradeForm.jsx]: Check out the form control and form group classes.

https://getbootstrap.com/docs/4.4/components/forms/

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by kusholhuq** [comment]
style 2

**Comment by kusholhuq** [comment]
empty div removed

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**